apps/prairielearn/python/prairielearn.py

```diff
@@ -745,38 +736,38 @@ def numpy_to_matlab(A, ndigits=2, wtype="f"):
     digits after the decimal and is formatted as "wtype" (e.g., 'f', 'g', etc.).
     """
     if np.isscalar(A):
-        A_str = "{:.{indigits}{iwtype}}".format(A, indigits=ndigits, iwtype=wtype)
-        return A_str
+        scalar_str = "{:.{indigits}{iwtype}}".format(A, indigits=ndigits, iwtype=wtype)
+        return scalar_str
```

Comment:
in this case the variable name `A_str` is actually better than `scalar_str`, because it is refering to the original variable `A`. 

It seems odd to me that bugbear (I assume that's what encouraged this change) would complain about the variable name `A_str` but not `A`. Does it only do local variables and not function arguments?

Reply:
I have temporarily disabled function arguments as that is a larger change. It can do both.

Reply:
In this case, I think its ok to leave all the current variable names. `A` is a common matrix name, and `A_str` is clearly the string version of it, it all makes sense.

If we are going to change it, it should be changed all at once to things like `matrix` and `matrix_str` so that we don't have weird stuff happening like `matrix_str` is the string version of `A` because that is just confusing.

Reply:
`A` is a scalar, vector, or matrix, so renaming it to `matrix` doesn't make sense to me. Also like you said, `A` is a common matrix name, so it gets confusing when it isn't.

Reply:
Ok I see. 

Then I think you should change the variable name `A` in this PR as well, even though it isn't required by the linter (yet).

Reply:
Agreed that we should rename `A`, but we can also get away without naming the resulting string at all, which IMO is a small improvement:

```py
return "{:.{indigits}{iwtype}}".format(A, indigits=ndigits, iwtype=wtype)
```

Reply:
Renamed `A` to `np_object`.

Reply:
looks good